unlock-app/src/components/interface/buttons/homepage/HomepageButton.js

```diff
@@ -41,11 +44,11 @@ export class HomepageButton extends React.Component {
           <TermsBox>
             <div>
               To access the dashboard you need to agree to our&nbsp;
-              <Link href="/terms">
+              <Link href={config.unlockStaticUrl + '/terms'}>
```

Comment:
We need a URL Builder at some point ;p

Reply:
Yes, but that's something for another PR ;)